x-pack/plugins/alerting/server/routes/find.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  IRouter,
+  RequestHandlerContext,
+  KibanaRequest,
+  IKibanaResponse,
+  KibanaResponseFactory,
+} from 'kibana/server';
+import { FindOptions } from '../../../alerting/server';
+import { LicenseState } from '../lib/license_state';
+import { verifyApiAccess } from '../lib/license_api_access';
+
+// config definition
+const querySchema = schema.object({
+  per_page: schema.number({ defaultValue: 20, min: 0 }),
```

Comment:
In the UI our default is 10, here it's 20.
Should these be the same?
Perhaps they should at least reside as constants side by side so wee can clearly see the difference?

Just thoughts, not necessarily something to address.

Reply:
Done.